### ISSUE TITLE
Is the project alive?

### ISSUE THREAD

**Issue by Khomyak-sibiryak** [Obscurix/Obscurix#2]
Hey.
There have been no updates for a long time and I'm worried the project is dead. I hope that I am wrong.

**Comment by madaidan** [comment]
It's on a hiatus for now. I've been too busy with other projects like Whonix.

**Comment by savchenko** [comment]
@madaidan , do you think there is a bit too much fragmentation?

I am currently working on https://github.com/stoptracking/playbook/ and it seems to me that there might be a fair amount of duplicated effort between "security-focused" systemd-based Linux distributions.

**Comment by madaidan** [comment]
> do you think there is a bit too much fragmentation?

I don't think so. Most projects I've seen have different goals/methods. For example, I've yet to find another hardened, live Linux distro using Tor, I2P, Freenet, etc. like Obscurix.

**Comment by savchenko** [comment]
> I've yet to find another hardened, live Linux distro using Tor, I2P, Freenet, etc. like Obscurix.

mm.... [Tails](https://tails.boum.org/)?

**Comment by madaidan** [comment]
Tails only supports Tor.

**Comment by savchenko** [comment]
> Tails only supports Tor.

_"We are very sad to announce that Tails 2.11 will be the last version to include I2P, an alternative anonymizing network._

_Maintaining software like I2P well-integrated in Tails takes time and effort and our team is too busy with other priorities._
_Unfortunately, we failed to find a developer outside of our team to maintain I2P in Tails."_

https://tails.boum.org/news/version_2.11/

**Comment by Zbergen-cli** [comment]
> I don't think so. Most projects I've seen have different goals/methods. For example, I've yet to find another hardened, live Linux distro using Tor, I2P, Freenet, etc. like Obscurix.

Why not create your own arch based distro?

**Comment by madaidan** [comment]
That's what Obscurix is.

**Comment by gjf** [comment]
With all respect taking into accout [Network issues](https://github.com/Obscurix/Obscurix/issues/3) are not reviewed for a year - I really think this project is close to dead.

**Comment by Zbergen-cli** [comment]
We'll have to make a fork.

**Comment by pscopic** [comment]
So tragic. This project really did have potential to be the arch based version of whonix. Debian it is then.

**Comment by username-is-not** [comment]
> Debian it is then.

Only issue I have w/debian is how to harden the kernel? *exactly
